.ci/pipelines/build.Jenkinsfile

```diff
@@ -37,37 +37,31 @@ pipeline {
                 stage('build') {
                     failFast true
                     parallel {
-                        stage("build and push operator image") {
+                        stage("build and push operator image and manifests") {
                             agent {
                                 label 'linux'
                             }
                             steps {
                                 sh '.ci/setenvconfig build'
-                                sh 'make -C .ci license.key TARGET=build-operator-multiarch-image ci'
+                                sh 'make -C .ci license.key TARGET=ci-release ci'
+                                sh 'make -C .ci yaml-upload'
                             }
                         }
-                        stage("build and push operator image in FIPS mode") {
+                        stage("build and push operator image and manifests in FIPS mode") {
                             agent {
                                 label 'linux'
                             }
+                            environment {
+                                ENABLE_FIPS="true"
+                            }
                             steps {
                                 sh '.ci/setenvconfig build'
-                                sh 'make -C .ci license.key TARGET=build-operator-multiarch-image ci ENABLE_FIPS=true'
+                                sh 'make -C .ci license.key TARGET=ci-release ci'
+                                sh 'make -C .ci yaml-upload'
```

Comment:
Could you please help me understand why we want to upload the manifests in fips mode?  Wasn't the intent of having a FIPS image/container simply to allow someone to do something like:

```
helm install elastic-operator -n elastic-system elastic/eck-operator --set=image.repository=docker.elastic.co/eck/eck-operator-fips --version 2.5.0
```

Or edit their eck-operator installation after install to update the image to be fips compliant?